plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/translators/TranslatorContext.cpp

```diff
@@ -498,7 +498,7 @@ void TranslatorContext::preRemoveEntry(
     // (which will guarentee the the itemsToRemove will be ordered such that the child prims will be
     // destroyed before their parents).
     auto iter = range_end;
-    itemsToRemove.reserve(itemsToRemove.size() + range_end - range_begin);
+    itemsToRemove.reserve(itemsToRemove.size() + (range_end - range_begin));
```

Comment:
Asserting in Debug because of invalid offset when range is empty; forcing (range_end - range_begin) to compute first solved the issue.